### PR TITLE
NY assembly votes

### LINKED ISSUE #1960: NY: No 2017 bills contain any Assembly votes
There are separate issues (#1384 , #1831) about missing roll calls.  This is about there being no 2017 Assembly votes at all in any bills.  I presume the other issues will remain after this is fixed, hence the new issue.

### PR DESCRIPTION
**Pull request by In-vincible**
resolves #1960,
Since #2185 worked on this issue earlier, I worked on top of it.
I was facing error with validation of texts.
I have commented the validation part in ny/utils.py, I have added separate commit for that I am not sure how to handle that part, please review this part and suggest what should be done.

### CODE REVIEW

File: openstates/ny/utils.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __repr__(self):
     @CachedAttr
     def text(self):
         text = self.scraper.get(self.url).text
-        self.urls_object.validate(self.name, self.url, text)
```
Comment (jamesturk) [inline openstates/ny/utils.py]: what error was this raising?

Comment (In-vincible) [reply openstates/ny/utils.py]: File "/media/bikram/0bb967d3-86e9-4eee-934c-c0fc3f76980d/home/bikram/openstates/openstates/ny/utils.py", line 148, in validate
    for validator in self._validators[name]:
AttributeError: 'Urls' object has no attribute '_validators'

**Comment by jamesturk** [comment]
OK, feel free to leave that commented out if it works without it, we need
to revisit that ny/utils stuff at some point but I'd rather get this
working first

**Comment by In-vincible** [comment]
@jamesturk its working after commenting it.

**Pull request merged**